fiftyone/utils/qwen3_8.py

```
@@ -0,0 +1,408 @@
+            answering, one of ``"low"``, ``"medium"`` or ``"xhigh"``. The
+            chat template turns this into a system instruction rather than a
+            hard limit. The model's own default is ``"xhigh"``; detection
+            needs no extended deliberation, and brief reasoning leaves more
+            of ``max_new_tokens`` for the answer
+        load_in_4bit (True): whether to load the language model with 4-bit
+            NF4 quantization on GPU; the vision encoder stays bf16. The full
+            bf16 weights are 55.6 GB. Set to False to load bf16 on hardware
+            that can hold it
+    """
+
+    def __init__(self, d):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)